src/cloud_client/dropbox/parameters/upload.rs

```diff
@@ -0,0 +1,21 @@
+use derive_builder::Builder;
```

Comment:
nit: I personally like https://github.com/idanarye/rust-typed-builder more as it has compile-type checks.